package/whiteboard-hypercard/package

```diff
@@ -1,32 +1,34 @@
 #!/usr/bin/env bash
-# Copyright (c) 2021 The Toltec Contributors
+# Copyright (c) 2023 The Toltec Contributors
 # SPDX-License-Identifier: MIT
 
 pkgnames=(whiteboard-hypercard)
 pkgdesc="Real-time collaboration, drawing or whiteboarding"
 url=https://github.com/fenollp/reMarkable-tools
-pkgver=0.3.4-1
-timestamp=2021-10-25T15:45Z
+pkgver=0.3.7-1
+timestamp=2023-10-11T12:57Z
 section="drawing"
 maintainer="Pierre Fenoll <pierrefenoll@gmail.com>"
-license=Apache-2.0
+license=CC-BY-NC-ND
 installdepends=(display)
 flags=(patch_rm2fb)
 
-image=rust:v2.2
+image=rust:v3.1
 source=(
-    https://github.com/fenollp/reMarkable-tools/archive/v0.3.4.zip
+    https://github.com/fenollp/reMarkable-tools/archive/v0.3.7.zip
     whiteboard-hypercard.draft
 )
 sha256sums=(
-    b343e1b4af9e0bf247ff2fab8331648ddd89941fb5c0ea9a45417d3bb4b20193
+    642fd954ec4f9a1d132b10cc7f7dfbee3467e9c08b1253cc32d1e372178d168b
     SKIP
 )
 
 build() {
-    cd marauder
+    pushd marauder
```

Comment:
I wouldn't recommend using pushd/popd here for a single directory change. This is just a recommendation though, I'm still going to approve the PR after it's tested.